assets/scss/components/_bounds.scss

```diff
@@ -13,12 +13,12 @@ padding, and sets a max-width.
 	box-sizing: border-box;
 	margin-left: auto;
 	margin-right: auto;
-	max-width: $bounds-wide;
+	max-width: $bounds;
```

Comment:
There's still a [`bounds--wide` class](https://github.com/meetup/meetup-web-components/pull/132/files/5173cbacc24b7998d180728ef8449455e3f00127#diff-887b5fa95ea8cd21b4b4f2d18e1ae9d2R23). But I don't think our `Bounds` components supports that as a prop yet.